src/app/pages/news/11.2/04-engine-cockpit.md

```diff
@@ -1,6 +1,13 @@
 ## Engine Cockpit {#engineCockpit}
 
-Blah Engine Cockpit blah
+We added awesome new world-class monitoring, user management, operations, and configuration features to the Engine Cockpit.
+
+- **Configuration**: Use the new **System/SSL** view to manage your certificates. The new **System/Config File Editor** helps you view and edit any configuration file of Axon Ivy. Press CTRL-SPACE to get content assistance. 
+- **User Management**: The user details view now shows you the Substitutes and Absences of a user. The security identity provider configuration view has been reimplemented with a new LDAP browser and Azure AD browser.
+- **Operations**: Restart the Axon Ivy Engine with the Engine Cockpit. Merge security contexts and move applications to another security context helps you migrate your Engine from LTS 8.
```

Comment:
The second sentence is hard to read. Maybe something like this?

Merge and move capabilities of applications and their security contexts simplify the migration from LTS 8.